[quant][test] Split TestQuantizeScript to two TestCase

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#36354 [quant][test] Split TestQuantizeScript to two TestCase**

Summary:
TestQuantizeScript is splitted to
- TestQuantizeScriptJitPasses
- TestQuantizeScriptPTSQOps (post training static quantization ops)

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

Differential Revision: [D20956731](https://our.internmc.facebook.com/intern/diff/D20956731)

jerryzh168 committed:
[quant][test] Split TestQuantizeScript to two TestCase

Summary:
TestQuantizeScript is splitted to
- TestQuantizeScriptPasses
- TestQuantizeScriptPTSQOps (post training static quantization ops)

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

supriyar commented on test/quantization/test_quantize_script.py:
```diff
@@ -50,1178 +50,1345 @@
-class TestQuantizeScript(JitTestCase):
-    def test_insert_observers(self):
-        class M(torch.nn.Module):
+class TestQuantizeScriptPasses(JitTestCase):
```
Can we call this TestQuantizeScriptJitPass instead? Or just TestQuantizedJitPasses

jerryzh168 replied:
sure, will rename to JitPasses

jerryzh168 committed:
Update on "[quant][test] Split TestQuantizeScript to two TestCase"


Summary:
TestQuantizeScript is splitted to
- TestQuantizeScriptPasses
- TestQuantizeScriptPTSQOps (post training static quantization ops)

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][test] Split TestQuantizeScript to two TestCase"


Summary:
TestQuantizeScript is splitted to
- TestQuantizeScriptJitPasses
- TestQuantizeScriptPTSQOps (post training static quantization ops)

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][test] Split TestQuantizeScript to two TestCase"


Summary:
TestQuantizeScript is splitted to
- TestQuantizeScriptJitPasses
- TestQuantizeScriptPTSQOps (post training static quantization ops)

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][test] Split TestQuantizeScript to two TestCase"


Summary:
TestQuantizeScript is splitted to
- TestQuantizeScriptJitPasses
- TestQuantizeScriptPTSQOps (post training static quantization ops)

Test Plan:
.

Reviewers:
.

Subscribers:

Tasks:

Tags:

Differential Revision: [D20956731](https://our.internmc.facebook.com/intern/diff/D20956731)

[ghstack-poisoned]
